scripts/node/README.md

```diff
@@ -0,0 +1,37 @@
+# Steps to benchmark node scanning 
+
+## Pre steps:
+
+1.Make sure docker is installed on your local laptop
+2.execute
+```
+docker run -it --rm registry.access.redhat.com/ubi8/ubi
```

Comment:
A docker run alone might work now, but only because the image has a default entrypoint which is bash.
How about we make this more explicit?
```suggestion
docker run -it --rm --entrypoint /bin/bash registry.access.redhat.com/ubi8/ubi
```